cpp/bench/ann/src/raft/raft_benchmark.cu

```diff
@@ -215,6 +215,21 @@ void parse_build_param(const nlohmann::json& conf,
   }
 }
 
+AllocatorType parse_allocator(std::string mem_type)
+{
+  if (mem_type == "device") {
```

Comment:
Can you also update the Ann bench param tuning guide to add the men types?

Reply:
updated